src/Umbraco.Core/Trees/Tree.cs

```diff
@@ -45,7 +45,7 @@ public Tree(int sortOrder, string applicationAlias, string group, string alias,
         /// </summary>
         public Type TreeControllerType { get; }
 
-        internal static string GetRootNodeDisplayName(ITree tree, ILocalizedTextService textService)
+        public static string GetRootNodeDisplayName(ITree tree, ILocalizedTextService textService)
```

Comment:
Similarly here - have made this public such that its accessible to `Umbraco.Web.BackOffice`.  We could use `InternalsVisibleTo` here if it's preferred.